tests/e2e/data/do-api/test_do_dataset_integration.py

```diff
@@ -118,18 +117,17 @@ def test_can_upload_a_dataframe_and_wait_for_completion(self):
         sample = StringIO(CSV_SAMPLE_REDUCED)
         df = pandas.read_csv(sample)
 
-        dataset = self.bq_user_dataset.name(unique_table_name) \
-                                      .column(name='id', type='INT64') \
-                                      .column('geom', 'GEOMETRY') \
-                                      .ttl_seconds(30)
+        dataset = self.do_dataset.name(unique_table_name) \
+            .column(name='id', type='INT64') \
+            .column('geom', 'GEOMETRY') \
+            .ttl_seconds(30)
         dataset.create()
         status = dataset.upload_dataframe(df)
 
         self.assertIn(status, ['success'])
 
     def test_can_download_to_dataframe(self):
-        result = self.bq_user_dataset.name('census_tracts_american_samoa') \
-                                     .download_stream()
+        result = self.do_dataset.name('census_tracts_american_samoa').download_stream()
```

Comment:
Note to myself: I'd need to review creds to test against in staging/prod, set up that table and write a few lines in the tests readme